bower.json

```diff
@@ -10,8 +10,6 @@
     "ember-resolver": "~0.1.20",
     "jquery": "1.11.3",
     "loader.js": "ember-cli/loader.js#3.4.0",
-    "qunit": "~1.20.0",
```

Comment:
Sure, I think this PR is still fine to go through in its current state. 

It's just then a case of updating the addon to Ember CLI 2.12.0 to remove the addon's dependency on bower packages, right?